app/ui/src/app/store/step/step.store.ts

```diff
@@ -94,10 +93,15 @@ export class StepStore {
       description: "Sends a message to the integration's log",
       configuredProperties: undefined,
       properties: {
-        message: {
+        expression: {
           type: 'string',
```

Comment:
Might be worth changing this to `textarea`?  Then it'll be a multiline input.  Have a look at how the advanced filter step is set up for details.

Reply:
Thanks! I will.